stock_warehouse_flow/views/stock_warehouse_flow.xml

```diff
@@ -0,0 +1,189 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2022 Camptocamp SA
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo>
+
+  <record id="stock_warehouse_flow_view_form" model="ir.ui.view">
+    <field name="name">stock.warehouse.flow.form</field>
+    <field name="model">stock.warehouse.flow</field>
+    <field name="arch" type="xml">
+      <form string="Warehouse Flow">
+        <sheet>
+          <widget
+                        name="web_ribbon"
+                        title="Archived"
+                        bg_color="bg-danger"
+                        attrs="{'invisible': [('active', '=', True)]}"
+                    />
+          <div class="oe_title">
+              <label for="name" class="oe_edit_only" />
+              <h1><field name="name" placeholder="Flow Name" /></h1>
+          </div>
+          <group>
+            <group name="info" colspan="4" col="4">
+              <field
+                                name="warehouse_id"
+                                attrs="{'readonly': [('delivery_route_id', '!=', False)]}"
+                            />
+              <field name="sequence" />
+              <field name="active" invisible="1" />
```

Comment:
company_id field is missing. It is required to be part of the view as you defined check_company on the field

Error while validating view:

Field company_id used in field out_type_id default domain ([('company_id', 'in', [company_id, False])]) must be present in view but is missing.